isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -367,28 +371,30 @@ Die Authentifizierung wird von der eigentlichen fachlichen Anfrage getrennt, so
 [[aufruf-service-gateway-mit-authentication-gateway]]
 ===== Direkte Authentifizierung am IAM-Service
 
-Die folgende Abbildung zeigt die neue System-Architektur, die bei Aufrufen von externen Anwendungen zum Einsatz kommt.
-Für die Authentifizierung einer externen Anwendung existiert ein eigener Servicepunkt zum _Authentication-Gateway_, der zuerst aufgerufen werden muss und ein Access-Token zurückliefert.
-Anschliessend wird die fachliche Anfrage (und alle weiteren) zusammen mit diesem Access-Token über das _Service-Gateway (SGW)_ an das Anwendungssystem geschickt.
+Die folgende Abbildung zeigt die zukünftig intendierte System-Architektur, die bei Aufrufen von externen Anwendungen zum Einsatz kommen soll.
 
 .Authentifizierung am Authentication-Gateway
 [id="image-servicegateway-plus-authgateway",reftext="{figure-caption} {counter:figures}"]
 image::isy-security:konzept/ServiceGateway_plus_AuthGateway.dn.svg[align="center"]
 
-Damit ein Client einer externen Anwendung auf die Ressourcen einer Anwendung zugreifen kann, benötigt er ein gültiges Access-Token.
-Dieses Token muss sich der Client zuerst beschaffen, indem er sich beim _IdentityAccessManagement-Service (IAM-Service)_ authentifiziert.
-Mit der Einführung des _Authentication-Gateways_ werden somit alle Anfragen mit Authentifizierungsdaten über einen separaten Kommunikationskanal geleitet.
-Damit kommt keine mit der Authentifizierung involvierte Komponente mit späteren Anwendungsanfragen in Kontakt und umgekehrt.
+In der Abbildung sind die folgenden Ablaufschritte dargestellt:
+
+. Für die Authentifizierung einer externen Anwendung existiert ein eigener Servicepunkt, das _Authentication-Gateway_. 
+An diesen Servicepunkt werden von der extenen Anwendung die Authentifizierungsdaten übergeben.
+. Das _Authentication-Gateway_ leitet die Authentifizierungsdaten an den _IAM-Service_ weiter.
+. Der _IAM-Service_ prüft die Authentifizierungsdaten und liefert dem  _Authentication-Gateway_ ein Access-Token zurück.
+. Das _Authentication-Gateway_ übergibt das Acess-Token der externen Anwendung.
+. Die externe Anwendung kann dieses Access-Token speichern und es anschließend für _mehrere_ fachliche Anfragen an das Anwendungssystem (über das _Service-Gateway_) verwenden.
+. Für jede dieser fachlichen Anfragen wird das jeweils mitgelieferte Access-Token validiert. Das aufgerufene Anwendungssystem prüft hierzu dessen Signatur, dessen Gültigkeit und die im Access-Token hinterlegten Berechtigungen,
```

Comment:
Zwei Sätze in einer Zeile. Bitte trennen.

Reply:
erledigt